### ISSUE TITLE
Call merge/conference Android (Self-Managed) and iOS 

### ISSUE THREAD

**Issue by AbdulBsit** [react-native-webrtc/react-native-callkeep#768]
# Question

- [Yes] I've checked the [example](https://github.com/react-native-webrtc/react-native-callkeep/tree/master/example) to reproduce the issue.

- Platforms need to be addressed:
  [Yes] Android
  [Yes] iOS

## Description
Is callkeep, allow conference in iOS and Android(Self Managed), suppose i have 3 calls in parallel, 2 are on hold,
scenarios are 
-  merge one call, 
-  more than one, 
- or all call at once
- 
How to do this in with this library for iOS and Android (self managed mode)
In case this involves native code, suggest me ways, how can we perform merge call scenarios
